src/common/internalLink/InternalLink.tsx

```diff
@@ -7,10 +7,12 @@ import styles from './internalLink.module.scss';
 type Props = {
   color?: 'standard' | 'critical' | 'brand';
   underlined?: boolean;
+  icon?: JSX.Element;
 } & LinkProps;
 
 const InternalLink: React.SFC<Props> = ({
   color = 'brand',
+  icon,
```

Comment:
I guess the icon here is redundant since it is not rendered.

Reply:
it's rendered in the sidebar

Reply:
I mean here in the return value of this InternalLink component it is not used anywhere! 
```jsx
return (
    <Link
      {...props}
      className={classNames(styles.internalLink, styles[color], {
        [styles.underlined]: underlined,
      })}
    >
      {children}
    </Link>
  );
```

Reply:
You're right :). Fixed.